src/Surfnet/StepupMiddleware/ApiBundle/Controller/RaListingController.php

```diff
@@ -77,10 +75,9 @@ public function getAction(Request $request, $identityId)
 
     /**
      * @param Request $request
-     * @param Institution $actorInstitution
      * @return JsonCollectionResponse
      */
-    public function searchAction(Request $request, Institution $actorInstitution)
+    public function searchAction(Request $request)
```

Comment:
Please clean up unused imports in this class.

Reply:
It looks like there are no unused imports?